client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -17,28 +17,68 @@ class GSuiteCancellationFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_purchases_gsuite_remove_purchase_learn_more_click' );
 	};
 
+	getAccessMessage = () => {
+		const { googleSubscriptionStatus, purchase, translate } = this.props;
+		const { meta: domainName, productSlug } = purchase;
+
+		const googleMailService = getGoogleMailServiceFamily( productSlug );
+
+		if ( [ 'suspended', '' ].includes( googleSubscriptionStatus ) ) {
```

Comment:
@daledupreez I am not sure which option would be better here, in case 'googleSubscriptionStatus' is empty string. I would say it would be better to fall into "immediately". As seems to me that having no subscription on Google, would not allow them to keep accessing it.

I also think that this might be rare case, I am not sure if that situation could end in this status, at this point in the code.

Reply:
Did you check what happens when this status is an empty string? As far I can see, we will now show [a placeholder](https://github.com/Automattic/wp-calypso/blob/f41f3f3e2ab72de22ec94452976f9cea321fc625/client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx#L78).

Reply:
I take that back - you left a screenshot showing what happens in that case:

![Screenshot](https://user-images.githubusercontent.com/5689927/132024174-e6cc6aba-8e45-4492-98c8-0edd09dbe701.png)

I'm not convinced that displaying that gray rectangle is the right approach though, it looks like something is broken. I would default to always showing the 'immediately' variant in that case.

Reply:
Removed placeholder :)